prod-devel-rcar.yaml

```diff
@@ -65,8 +65,7 @@ common_data:
     - [SOC_FAMILY, "%{SOC_FAMILY}"]
 
     # Add systemd configuration
-    - [DISTRO_FEATURES_append, " systemd"]
```

Comment:
This removal may introduce issues. We have few recipes (gles-user-module, optee_client) in meta-xt-rcar and meta-renesas that checks presence of `systemd` in `DISTRO_FEATURES`.
May be it's better to do not remove this line.

Reply:
Sorry, my previous comment is not valid. "systemd" is appended in `init-manager-systemd.inc`